changelog.md

```diff
@@ -3,18 +3,33 @@
 # Features/Improvements
```

Comment:
Worth mentioning the ongoing l10n/i18n? It's a pretty small number of areas atm, but folks would probably be glad to hear that it's at least in progress.

Reply:
Locale selection isn't in the dev branch, so there's nothing actually accessible to the user that's translated.  

I could add the selection back in if we want to, than the Chinese briefing we have would be accessible. The Enums I implemented are getting replaced with built-in language types, so I closed the original PR without merge.